common/types.ts

```diff
@@ -0,0 +1,97 @@
+// This file contains common types used through out the project.
+
+export interface book {
+    id: string;
+    isbn: string;
+    name: string;
+    imageUrl: string | null;
+    googleBooksId: string | null;
+    isCampusStoreBook: boolean;
+    campusStorePrice: number | null;
+    courses: course;
+    posts : post;
+    googleBook : googleBook | Object;
+  };
+  
+  export interface course {
```

Comment:
I think some of these interfaces should be imported from the prisma client. It looks like there is a mismatch between theses interfaces and our prisma model.

Reply:
Agreed. Working on that now because the Vercel build is failing.

Will push a commit excluding these types in a bit.

Reply:
Done. The code now uses Prisma types.